design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+    id: ID!
+    "Name of the list"
+    name: String!
+    "Description of the list"
+    description: String
+    "Items in the list"
+    items: [RequisitionListItemInterface]
+    "Number of items in list"
+    items_count: Int!
+    "Latest Activity"
+    updated_at: String
+}
+
+"Interface type for Requisition List Item"
+interface RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+}
+
+type SimpleRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+}
+
+type VirtualRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+}
+
+type DownloadableRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+    "DownloadableProductLinks defines characteristics of a downloadable product"
+    links: [DownloadableProductLinks]
+    "DownloadableProductSamples defines characteristics of a downloadable product"
+    samples: [DownloadableProductSamples]
+}
+
+type BundleRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+    "selected bundle options"
+    bundle_options: [SelectedBundleOption]!
+}
+
+type ConfigurableRequisitionListItem implements RequisitionListItemInterface {
+    "Unique Identifier of Requisition List Item"
+    id: ID!
+    "Product"
+    product: ProductInterface!
+    "Quantity added"
+    qty: Float!
+    "custom Option selected"
+    customizable_options: [SelectedCustomizableOption]
+    "Configurable options selected"
+    configurable_options: [SelectedConfigurableOption]
+}
+
+type Mutation {
+    "Create Empty Requisition List"
+    createRequisitionList(
+        "name for the list"
+        name: String!,
+        "description For the list"
+        description: String
+    ): RequisitionList
+
+    "Rename a requisition list"
+    renameRequisitionList(
+        "unique Id of requisition list"
+        id: ID!,
+        "new name for list"
+        name: String!,
+        "new description For the List"
+        description: String!
```

Comment:
`createRequisitionList` has `description` as optional, but `renameRequisitionList` has it as a required arg.

These should be consistent

Reply:
Checked with luma storefront, description is optional, shall I make it optional. @mbrinton01

Reply:
It is optional and should be that way in all actions/places.